desktop/apps/consignments2/components/submission_flow/index.js

```diff
@@ -28,28 +34,37 @@ const mapStateToProps = (state) => {
   const {
     submissionFlow: {
       currentStep,
+      isMobile,
       steps
     }
   } = state
 
+  const describeWorkComponent = isMobile ? DescribeWorkMobile : DescribeWorkDesktop
```

Comment:
I think the approach you took is clearer. No need to cram a bunch of responsive code into a single component when you can separate.